packages/react/src/patterns/sub-patterns/TableOfContents/TableOfContents.js

```diff
@@ -63,23 +99,6 @@ const TableOfContents = ({ menuItems, children, menuLabel, theme }) => {
     return elesInView;
   };
 
-  /**
-   * Set selected id & title
-   *
-   */
-  const setSelectedItem = () => {
-    const elems = getElemsInView();
-    const id = elems[0] || menuItems[0].id;
-    const filteredItems = menuItems.filter(menu => {
-      if (id !== 'undefined') {
-        return menu.id === id;
-      }
-    });
-    const title = filteredItems[0].title;
-    setSelectedId(id);
-    setSelectedTitle(title);
-  };
```

Comment:
This was probably fine where it was before.

Reply:
@jeffchew If  I am trying to move this to the same place .. I am getting the below error
<img width="692" alt="Screen Shot 2020-02-03 at 2 22 11 PM" src="https://user-images.githubusercontent.com/12287070/73683762-da4dfb00-4690-11ea-9b26-3b983ecb1f2e.png">

Reply:
Never mind .... updated that and move back